### DISCUSSION TITLE
canActivateAR incorrectly returning false on iOS 16 chrome/firefox

### DISCUSSION THREAD

**Discussion by CameronCrane** [google/model-viewer#3861] [Q&A]
### Description

It seems that with the release of iOS 16, the canActivateAr method incorrectly returns false on iOS 16/chrome and iOS 16/Firefox, even if manually setting `arModes="webxr scene-viewer quick-look"`. On the same device (using an iPhone 13 Pro in this case), it returns true _only_ when using Safari. 

#### Live Demo
Here are some snippets of our implementation.

```
public get canActivateAR() {
      return this[$modelViewer].value
        ? this[$modelViewer].value.canActivateAR
        : false;
  }
```

```
 private [$onArButtonClick]() {
      if (this.canActivateAR) {
        alert('can activate AR')
        return this[$modelViewer].value?.activateAR();
      }
      alert('cannot activate AR');
  }
```

### Version
- model-viewer: v1.10.1 (currently unsure if this also applies to the newest version)

### Browser Affected
- [X] Chrome
- [ ] Edge
- [X] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [X] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
- [ ] WebXR
- [ ] SceneViewer
- [X] QuickLook

**Comment by jediskej** [comment]
I'm having the same problem on Android. Any solutions?

**Reply by milesgreen** [reply]
Any more details @jediskej? The iOS16 change shouldn't have affected Android. Might be worth filing a separate issue.

**Reply by jediskej** [reply]
for the past 2 days the model viewer AR hasnt been working for me. models that I could view before in AR now have very long load times, and when they finish loading, the "view in your space" button disappears. I just tried using the basic astronaut model and had the same issue.

**Reply by elalish** [reply]
@jediskej Can you file an issue on this with the detail requested in the template? Also, was this a due to change in model-viewer version or OS/browser version update? And what AR mode? There was a recent bug that rolled out to Scene Viewer, but it doesn't affect WebXR mode.

**Reply by elalish** [reply]
#3849

**Comment by elalish** [comment]
Indeed, this is an iOS regression and was worked-around in v2.0: https://github.com/google/model-viewer/pull/3784

**Reply by milesgreen** [reply]
@CameronCrane - I'd be keen to know if upgrading to v2.0+ doesn't fix this for you. 

If you're not ready to upgrade to v2.0+, you could fork your current version and apply the changes made in the PR mentioned above.

**Reply by milesgreen** [reply]
I should say, there's a lot of good stuff in v2.0+, so you should probably be heading in that direction.
